src/backend/storage/utils/btree.h

```diff
@@ -0,0 +1,63 @@
+#include "page.h"
+#include <cstdint>
+#include <vector>
+
+#ifndef BTREE_H
+#define BTREE_H
+
+class BTreeNodeHeader {
```

Comment:
Name this NodeHeader and add this under namespace BTree

Reply:
how to add under namespace BTree